specs/interop/derivation.md

```diff
@@ -0,0 +1,22 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+  - [Deposit Context](#deposit-context)
+- [Security Considerations](#security-considerations)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview 
+
+This is an experimental section and may be changed in the future. It is not required
```

Comment:
We do want this for the initial release of interop, what I think we should cut from the initial mainnet release is the ability for deposits to trigger executing messages, we can follow up with that functionality